src/frontend/komponenter/Felleskomponenter/PersonInformasjon/PersonInformasjon.tsx

```diff
@@ -23,12 +23,23 @@ const StyledDropdownMeny = styled(Dropdown.Menu)`
     width: 20ch;
 `;
 
-const FlexBox = styled.div`
-    display: flex;
-    align-items: center;
-    gap: 0.25rem;
+const HeadingUtenOverflow = styled(Heading)`
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
 `;
 
+const Skillelinje: React.FC<{ erHeading?: boolean }> = ({ erHeading = false }) => {
+    if (erHeading) {
+        return (
+            <Heading level="2" size="medium" as="span">
+                |
+            </Heading>
+        );
+    }
+    return <BodyShort>|</BodyShort>;
+};
```

Comment:
Wrapper man med `Heading` og `BodyShort` for å styre størrelsen på skillelinja?

Reply:
Jepp... Syns ikke egentlig det er en så fin løsning, men videreførte bare sånn det var fra før 🤷‍♂️